website/index.html

```diff
@@ -193,65 +202,79 @@
           <div class="offset-md-2 col-md-8 ">
             <div class="row">
               <div class="col-12 header-text wrapped-box">
-                <p class="text-left"><span>HackPSU Has Been Cancelled</span></p>
-                <p class="text-left"><span>Due to COVID-19</span></p>
-                  <p class="text-left"><span>Click <a href="#message">Here</a> For More Info</span></p>
+                <p class="text-left"><span>November 6-8</span></p>
+                <p class="text-left"><span>Penn State University</span></p>
+                <p class="text-left"><span>Virtual!</span></p>
               </div>
             </div>
           </div>
         </div>
 
-        <div class="row cta">
-          <div class="offset-md-2 col-lg-8 col-md-10 col-sm-12 register-btn">
-            <a href="https://app.hackpsu.org/login">
-              <div class="header-text text-center">
-                <p class="text-center">LOGIN</p>
-              </div>
-            </a>
+        <div class="row cta" style="margin: 3vh">
+          <div class="offset-md-2 col-lg-8 col-md-10 col-sm-12 register-btn btn-with-planet" >
+            <div class="row">
+              <a href="https://app.hackpsu.org/login">
+                <img src="./assets/images/planet6small.svg" class="planet"/>
+              </a>
+              <a href="https://app.hackpsu.org/login">
+                <div class="header-text text-center">
+                  <p class="text-center">LOGIN</p>
```

Comment:
Planets on the buttons do a weird second bounce when you leave the button

Reply:
Hmm it's not happening on my end